packages/components/src/Accordion/useAccordion.js

```diff
@@ -216,10 +216,12 @@ export function useAccordion({ id, visible: visibleProp }) {
 		set,
 	} = useAccordionContext();
 
-	const visible = getIsVisible(id);
+	const isWithinContext = set !== noop;
```

Comment:
Probably not the best way to check 🤔.

Reply:
Definitely not, it leaks a huge implementation detail from `useAccordionContext`. Can we just default `set` to `null` instead?

Reply:
I have a better idea :)! Pushing it up soon